yml/cdb_lsstcomcam.yaml

```diff
@@ -1227,23 +1227,20 @@ tables:
   - name: eff_time
     "@id": "#ccdvisit1_quicklook.eff_time"
     datatype: float
-    description: Effective exposure time.
+    description: Effective exposure time calculated from PSF sigma, sky background, and zero point.
     ivoa:unit: s
   - name: eff_time_psf_sigma_scale
     "@id": "#ccdvisit1_quicklook.eff_time_psf_sigma_scale"
     datatype: float
     description: Effective exposure time, PSF sigma scale.
```

Comment:
Why were the units removed?

Reply:
It might make sense to change the description to be "Scale factor for effective exposure time based on PSF sigma".

Reply:
100%.  Also perhaps a better name pattern would be `eff_time_scale_psf_sigma` (etc.) -- it's an "effective time scale [factor]" due to "PDF sigma"

Reply:
If you are interested in the original naming discussion surrounding these quantities...
https://lsstc.slack.com/archives/C01JTSDV7CH/p1706307463383399